ntoskrnl/io/iomgr/deviface.c

```diff
@@ -32,39 +32,21 @@ static
 NTSTATUS
 OpenRegistryHandlesFromSymbolicLink(IN PUNICODE_STRING SymbolicLinkName,
                                     IN ACCESS_MASK DesiredAccess,
-                                    IN OPTIONAL PHANDLE GuidKey,
-                                    IN OPTIONAL PHANDLE DeviceKey,
-                                    IN OPTIONAL PHANDLE InstanceKey)
+                                    IN BOOLEAN bCreate,
```

Comment:
Nit: In the realm of NT, Hungarian notation isn't used (nor accepted). You could simply rename it just as `Create`, `CreateNonExistent` or any name of you choice. Other than that, the PR is OK.

Reply:
Closing as superseded.